hooks/useNetworkStatus.ts

```
@@ -0,0 +1,131 @@
+    switch (type) {
+      case Network.NetworkStateType.WIFI:
+        return 'excellent';
+      case Network.NetworkStateType.CELLULAR:
+        return 'good';
+      case Network.NetworkStateType.ETHERNET:
+      case Network.NetworkStateType.BLUETOOTH:
+      case Network.NetworkStateType.VPN:
+        return 'good';
+      case Network.NetworkStateType.WIMAX:
+        return 'poor';
+      case Network.NetworkStateType.NONE:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)